docs/user_guide/mitigation.rst

```diff
@@ -15,13 +15,13 @@ Fairlearn contains the following algorithms for mitigating unfairness:
       - binary classification
       - regression
       - supported fairness definitions
-   *  - :code:`fairlearn.` :code:`reductions.` :code:`ExponentiatedGradient`
+   *  - :class:`fairlearn. reductions. ExponentiatedGradient <fairlearn.reductions.ExponentiatedGradient>`
```

Comment:
```suggestion
   *  - :class:`fairlearn. reductions. ExponentiatedGradient`
```

Reply:
This would mean that it's not a link, just like with the earlier comments on using `.` as a prefix. It's not pretty, but I'd rather have it link to the API docs, especially since this is an overview table.

Reply:
If the link is not there, there's something else which needs to be fixed in the config files to make sure sphinx finds the relevant objects to point to. And I think it does raise a warning whenever it can't find a link, and falls back to verbatim.

Reply:
Hmm, `intersphinx` doesn't have a lot of config options https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html and we just use the very basic ones through the `intersphinx_mapping` dict: https://github.com/fairlearn/fairlearn/blob/master/docs/conf.py

You're right about verbatim, but why not write it in a way that works in the few cases where it doesn't manage to produce the link?

I'll see if I can find some time to document a case in a simple way and report it to (inter-)sphinx.